src/squadGame/src/component.js

```diff
@@ -1,52 +1,20 @@
-import React, { Fragment, Component } from 'react';
+/* eslint-disable */
```

Comment:
remove this.

Reply:
@walkwel let's integrate the code first, and edit from there